### ISSUE TITLE
[Feature Request] completed-list improvements

### ISSUE THREAD

**Issue by strangeman** [sachaos/todoist#29]
Thank you for the great tool! 
Main reason because I use the Todoist CLI client is the confusing `Completed tasks` interface. Your client almost perfectly fits my needs. But I suggest some improvements:

1) Shortcut for `completed-list` command (`cl`, for example).
2) `--filter` option. I often want to look what I did today or yesterday.

Currently, I added this features with bash aliases for myself, but I can add it to the code if this features really useful for others.

**Comment by strangeman** [comment]
My aliases FYI:
```
alias todo='todoist s && todoist'
alias todo-cl='todoist s && todoist completed-list'
alias todo-cl-today='todoist s && todoist completed-list | grep `date +%y/%m/%d`'
alias todo-cl-yesterday='todoist s && todoist completed-list | grep `date +%y/%m/%d --date="day ago"`'
```

**Comment by sachaos** [comment]
LGTM 👍

**Comment by sachaos** [comment]
This issue solved by #31 .
Thanks @strangeman :tada:
